pkgs/development/python-modules/tornado/5.nix

```diff
@@ -3,12 +3,13 @@
 , buildPythonPackage
 , fetchPypi
 , isPy27
+, pythonAtLeast
 }:
 
 buildPythonPackage rec {
   pname = "tornado";
   version = "5.1.1";
-  disabled = isPy27;
+  disabled = isPy27 || pythonAtLeast "3.10";
```

Comment:
```suggestion
  disabled = pythonOlder "3.5" || pythonAtLeast "3.10";
```

Reply:
I would rather not readd support for python2 that way.

Reply:
I took that from https://pypi.org/project/tornado/ on the bottom left.

Reply:
Check again, this is tornado 5.x. https://pypi.org/project/tornado/5.1.1/

Reply:
Oh, you're right.